.github/workflows/actions.yml

```diff
@@ -13,9 +13,8 @@ permissions:
 jobs:
   contracts:
     name: Contracts
-    uses: multiversx/mx-sc-actions/.github/workflows/contracts.yml@v2.3.5
+    uses: multiversx/mx-sc-actions/.github/workflows/contracts.yml@v3.0.0
```

Comment:
No version for tests runner, thus the latest available will be used. Seems fine :+1: 

https://github.com/multiversx/mx-sc-actions/blob/main/.github/workflows/contracts.yml#L23